renovate-changesets/index.ts

```diff
@@ -17,6 +17,8 @@ async function main() {
     required: false,
   });
 
+  const includeDevDependencies = core.getBooleanInput('includeDevDependencies', { required: false });
```

Comment:
so that we don't break things

```suggestion
  const excludeDevDependencies = core.getBooleanInput('excludeDevDependencies', { required: false });
```